src/functions/template-tags/link.php

```diff
@@ -357,19 +357,45 @@ function tribe_get_event_link( $post_id = null, $full_link = false ) {
 	 * @return string $html
 	 */
 	function tribe_get_event_website_link( $event = null, $label = null ) {
-		$url = tribe_get_event_website_url( $event );
+		$url    = tribe_get_event_website_url( $event );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: 25a3f2a14d8068a53de5774f262dd70a</sub>